toc/working-groups/app-runtime-interfaces.md

```diff
@@ -94,6 +94,11 @@ areas:
     github: plamen-bardarov
   - name: Ramon Makkelie
     github: ramonskie
+  bots:
+  - name: Cloud Foundry Buildpacks Team Robot
+    github: cf-buildpacks-eng
+  - name: ARI WG Git Bot
+    github: ari-wg-gitbot
```

Comment:
Both bots have already access to the buildpack repos because they are wg-scoped bots. Access is granted by team [wg-app-runtime-interfaces-bots](https://github.com/orgs/cloudfoundry/teams/wg-app-runtime-interfaces-bots). See also [rfc-0014-github-teams-and-access.md
](https://github.com/cloudfoundry/community/blob/main/toc/rfc/rfc-0014-github-teams-and-access.md)

What would make sense: move `cf-buildpacks-eng` from WG scope to area scope.